### ISSUE TITLE
Cannot open link

### ISSUE THREAD

**Issue by W4RH4WK** [snapcrafters/discord#1]
I have Discord installed as Snap on Ubuntu 16.04, including `snapd-xdg-open`. Yet I am unable to directly open links from Discord. I have to copy and paste them manually into the browser.

Please let me know whether this is already a known issue, or this is wrong place to report such an issue.

```
$ sudo apt install snapd-xdg-open
Reading package lists... Done
Building dependency tree       
Reading state information... Done
snapd-xdg-open is already the newest version (0.0.0~16.04).
0 upgraded, 0 newly installed, 0 to remove and 77 not upgraded.

$ sudo snap install discord
snap "discord" is already installed, see "snap refresh --help"
```

Edit: Running `xdg-open` from the commandline works fine.

**Comment by Ads20000** [comment]
Links are broken in all snaps at the moment, [a `snapd` update broke `snapd-xdg-open`](https://bugs.launchpad.net/snapd/+bug/1705708) and the snappy developers are [still working on a replacement for `snapd-xdg-open`](https://github.com/snapcore/snapd/pull/3260) which will allow clicking on external links to work on all Linux desktops without requiring a separate package.

**Comment by Ads20000** [comment]
[Links are now fixed on Ubuntu](https://bugs.launchpad.net/snapd/+bug/1705708/comments/8). [Will be fixed on all distros when `snapd` 2.28 is released](https://github.com/snapcore/snapd/pull/3260#issuecomment-325560019).
